### PR TITLE
Add Coreum to chain registry

### PR DESCRIPTION
**Pull request by wojtek-coreum**

### CODE REVIEW

File: coreum/chain.json [hunk 1]
```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "coreum",
+  "status": "upcoming",
+  "network_type": "mainnet",
+  "website": "https://www.coreum.com",
+  "pretty_name": "Coreum",
+  "chain_id": "coreum-mainnet-1",
+  "bech32_prefix": "core",
```
Comment (JeremyParish69) [inline coreum/chain.json:9]: Please register this bech32 prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

**Comment by JeremyParish69** [comment]
Very thorough! Can I suggest a circular token logo?

File: coreum/chain.json (hunk 1 above)
Comment (wojtek-coreum) [reply coreum/chain.json:9]: Done: https://github.com/satoshilabs/slips/pull/1521

**Comment by wojtek-coreum** [comment]
Logo design is far beyond me

File: coreum/chain.json (hunk 1 above)
Comment (wojtek-coreum) [reply coreum/chain.json:9]: @JeremyParish69 

The other PR has been merged

**Pull request merged**